### PR TITLE
chore(PPDSC-2775): update accordion default styles

### PR DESCRIPTION
**Pull request by Xin00163**
PPDSC-2775

Update default tokens value highlighted in red.
Extend already existing codemod script for automatically keeping the current tokens in the user component.

**I have done:**
- [x] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2775-accordion-default-styles
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2775-accordion-default-styles/storybook

File: src/accordion/style-presets.ts [hunk 1]
```diff
@@ -24,6 +27,12 @@ export default {
       safariOutlineStyle: '{{outlines.safariOutlineStyleDefault}}',
     },
   },
+  accordionHeaderOld: {
+    __extends: 'accordionHeader',
+    base: {
+      backgroundColor: '{{colors.interface010}}',
```
Comment (mstuartf) [inline src/accordion/style-presets.ts:33]: Will the change to `iconColor` have any impact?

**Review by mstuartf — COMMENTED** [review]
@Xin00163 we also need to document this codemod here: https://github.com/newscorp-ghfb/newskit/blob/v7-breaking-changes/lib/codemod/README.md#v700

File: src/accordion/style-presets.ts (hunk 1 above)
Comment (Xin00163) [reply src/accordion/style-presets.ts:33]: Yeah good point. It's a bug but should allow users to revert back to the old behaviour

**Review by mutebg — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Review by Xin00163 — COMMENTED** [review]
Design has decided to remove `fontColorChange` transitionPreset from button, so the previous changes are removed now.

**Pull request merged**